examples/gno.land/p/samcrew/keccak256/sha3.gno

```diff
@@ -0,0 +1,236 @@
+// Copyright 2014 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package keccak256
```

Comment:
add a package level comment explaining that this code comes from x/crypto/sha3

Reply:
Hello  Thanks for your review :)
done here e4d08b52fb29054e54e64fc46c244aa758d491a8